docs/pages/installation.mdx

```diff
@@ -208,3 +208,24 @@ $ export arch=amd64 # '386' 'arm' on linux or 'amd64' for all distros
 $ curl https://get.gravitational.com/teleport-$version-$os-$arch-bin.tar.gz.sha256
 # <checksum> <filename>
 ```
+
+
+## Operating System Support
+
+Teleport is officially supported on the platforms listed below. It is worth noting
+that the open source community has been successful in building and running Teleport on
+UNIX variants other than Linux \[1].
+
+| Operating System | Teleport Client | Teleport Server |
+| - | - | - |
+| Linux v2.6+ | yes | yes |
```

Comment:
Go minimum requirements are 2.6.23 and above. For example 2.6.18 (RHEL/CentOS 5) won't work.

https://github.com/golang/go/wiki/MinimumRequirements